packages/clay-card/fixtures/file/CardFile.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+import ClayCard from '../../src/index';
+
+import ClayIcon from '@clayui/icon';
+import ClayLabel from '@clayui/label';
+import ClaySticker from '@clayui/sticker';
+
+import React from 'react';
+
+const CardFile = (props: any) => (
+	<ClayCard displayType="file">
+		<ClayCard.Header>
+			<div className="aspect-ratio-item aspect-ratio-item-center-middle aspect-ratio-item-fluid card-type-asset-icon">
+				<ClayIcon
+					spritemap={props.spritemap}
+					symbol="documents-and-media"
+				/>
+			</div>
+			<ClaySticker displayType="danger" position="bottom-left">
+				{'DOC'}
+			</ClaySticker>
+		</ClayCard.Header>
+		<ClayCard.Body>
+			<div className="autofit-col autofit-col-expand">
```

Comment:
Are these classes always required? Ideally we shouldn't require any other markup than the components we export for composition. Unless the markup is content inside the component, but this case seems more related to the positioning of the content specifically for card.

This comment applies to other instances like this as well, there are quite a few inside of "fixtures" components.

Reply:
I removed the most part of fixtures, only leaving a Clay Checkbox with state for reusing on stories.